internal/testutil/mocksvc/cluster_service.go

```diff
@@ -1,177 +0,0 @@
-// Code generated by mockery. DO NOT EDIT.
-
```

Comment:
should we also adjust the mockery configuration? surprised our CI check "Mock generation" did not catch this, but maybe it is because the interface has been removed.

Reply:
I think so, that file doesn't exist any more